plugins/reach/plugins/OpenCalais/batch/RuleEngine.php

```diff
@@ -155,7 +158,7 @@ protected function handleCuePoints(array &$cuePoints, $key, array $nameArray)
             }
         }
         $fullName = trim($fullName);
-        $cuePoints[] = array('title' => $fullName, 'startTime' => $startFrom, 'endTime' => $endTo);
+        $cuePoints[] = array('title' => $fullName, 'startTime' => $startFrom, 'endTime' => $endTo, 'partnerData' => $permid);
```

Comment:
@TalDubovKaltura  wouldn't it be easier here to pass all the properties with the exact names as on the KalturaCodeCuePoint object (text, startTime, entryId, partnerData), and then iterating over them like this:
for ($cuePoint as $prop => $value) {
       $cuPoint->$prop = $value;

}

Reply:
@hilak That logic is a year ago and I assume this because of the 'endTime' prop which is not relevant anymore. I removed it and added the logic you mentioned. CR changes: https://github.com/kaltura/server/pull/11403/commits/35deec50aa730b427a2eaa46d6306c1fd2850773